tests/testthat/test-use-tutorial.R

```diff
@@ -0,0 +1,26 @@
+context("test-use-tutorial")
+
+test_that("use_tutorial() checks its inputs", {
+  skip_if_not_installed("rmarkdown")
+
+  scoped_temporary_package()
+  expect_error(use_tutorial(), "no default")
+  expect_error(use_tutorial(name = "tutorial-file"), "no default")
```

Comment:
Suggest you add one test with an invalid `name`, since the docs claim that certain requirements are enforced.